More clerify README (#159)
aio-libs/aioelasticsearch: README.rst

.. image:: https://codecov.io/gh/aio-libs/aioelasticsearch/branch/master/graph/badge.svg
  :target: https://codecov.io/gh/aio-libs/aioelasticsearch

Getting Started

> ------------

Reviewer: Underlining should match the text length.

> Aioelasticsearch - the asynchronous python client wrap for elasticsearch, the client needs to be connected to the elasticsearch server.

Reviewer: ```suggestion
Aioelasticsearch - is an asynchronous elasticsearch python client wrapper.
```

Prerequisites
------------

> You can run Elasticsearch locally or in a docker.

Reviewer 2: `Running Elasticsearch server is required` sound better as for me

Run it with docker:

.. code-block:: shell

>     docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.0.0

Reviewer 2: There is no need for 9300

Please, lets stick with es 6.0.0 as for now

.. image:: https://codecov.io/gh/aio-libs/aioelasticsearch/branch/master/graph/badge.svg
  :target: https://codecov.io/gh/aio-libs/aioelasticsearch

Getting Started
---------------

> Aioelasticsearch - is an asynchronous elasticsearch python client wrapper.

Reviewer: ```suggestion
Aioelasticsearch — is an asynchronous elasticsearch python client wrapper.
```

Prerequisites
-------------

Running Elasticsearch server is required.

> Run it with docker (Linux/MAC):

Reviewer: MAC is Mandatory Access Control or Media Access Control. Linux is a kernel, not an OS.
```suggestion
Run it with docker (GNU/Linux or macOS):
```

    loop.run_until_complete(go())
    loop.close()

Running the tests
-----------------

> Pytests

Reviewer 2: What it means here?

.. code-block:: python

>     python -m pytest 

Reviewer 2: pytest tests

Reviewer: I'd put test path into `pytest.ini` rather than forcing the user to do this manually

> For TOX tests with different python install different python form *tox.ini* and enter *tox* in project directory.

Reviewer 2: Seems some typos here, it is little bit hard to read it

    loop.run_until_complete(go())
    loop.close()

Running the tests
-----------------

> Enter *pytest* and *tox* in project directory for run tests.

Reviewer: ```suggestion
Type *pytest* or *tox* in the project directory to run tests.
```
(still seems quite inaccurate — what does each one of those do? how do they differ?)
